### DISCUSSION TITLE
Question about the shape of the touch_grid plugin sensor.

### DISCUSSION THREAD

**Discussion by kevinqyh0827** [google-deepmind/mujoco#1706] [General]
Hi, there. I am trying the touch_grid plugin sensor. I noticed the "FOV", field of view, features of this plugin. I wonder if this makes the shape of this sensor only in a spherical shape? Or it is related to the shape of the related "site" and the shape of this sensor can be adjusted by modifying the shape of the "site".

Thanks so much!

**Comment by yuvaltassa** [comment]
This is documented here
https://github.com/google-deepmind/mujoco/tree/main/plugin/sensor#touch-grid
